apps/trustlab/src/payload/blocks/CollectionOverview.js

```diff
@@ -0,0 +1,58 @@
+function CollectionOverview(
+  slug,
+  imageURL,
+  collections,
+  {
+    fields: additionalFields = [],
+    minRows = 1,
+    maxRows = 3,
+    required = true,
+    titleLabel = "Title",
+    collectionLabel = "Resources",
+    hasLinkButton = true,
+    linkLabelDefaultValue = "Learn more",
+    ...others
+  },
+) {
+  const fields = [
+    {
+      name: "title",
+      label: titleLabel,
+      type: "text",
+      required: true,
+      localized: true,
+    },
+    {
+      name: "collections",
```

Comment:
```suggestion
      name: "items",
```

items from one or more collections